src/lightmap.cpp

```diff
@@ -832,6 +836,8 @@ void map::build_seen_cache( const tripoint &origin, const int target_z )
         }
     }
 
+    std::fill_n( &camera_cache[0][0], SEEX * MAPSIZE * SEEY * MAPSIZE, 0.0f );
```

Comment:
This is the one bit I'm not sure about.  If I take this out the camera cache never gets cleared, I can't figure out how the seen_cache gets cleared so I can duplicate that.

Reply:
Figured it out and fixed it.